debug_toolbar/apps.py

```diff
@@ -174,3 +175,37 @@ def check_panels(app_configs, **kwargs):
             )
         )
     return errors
+
+
+@register()
+def js_mimetype_check(app_configs, **kwargs):
+    """
+    Check that JavaScript files are resolving to the correct content type.
+    """
+    javascript_types = {
+        # Macs
```

Comment:
Maybe `Linux, macOS` ? Or `Unices` ?

Reply:
This section confuses me. I put those comments there because on my mac, I found that mimetypes was returning application/javascript, but the windows solution was to use text/javascript.

I couldn't reproduce what windows users would actually experience for this. Or at least not easily.

Reply:
I installed python on my windows machine, which was easier than expected. We should check for text/javascript, but we should recommend that users use a value of application/javascript instead. I'll clean this up to reflect that.